app/src/main/res/layout/topic_review_fragment.xml

```diff
@@ -1,18 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout
-  xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto">
-  <androidx.constraintlayout.widget.ConstraintLayout
+  xmlns:android="http://schemas.android.com/apk/res/android">
+  <data>
+    <import type="android.view.View"/>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.topic.review.TopicReviewViewModel"/>
+  </data>
+  <FrameLayout
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
-    <TextView
-      android:id="@+id/dummy_text_view"
-      android:layout_width="wrap_content"
+    android:layout_height="match_parent"
```

Comment:
Please update the alignment

Reply:
This is correct. Just check that I have removed some lines above because of which it might be confusing.